### DISCUSSION TITLE
5 dependencies

### DISCUSSION THREAD

**Discussion by ghost** [MarkusJx/node-java-bridge#64] [Ideas]
Hello. Am I correct in assuming that dependencies: 
```json
    "dependencies": {
        "chalk": "^5.2.0",
        "glob": "^8.1.0",
        "ora": "^6.1.2",
        "typescript": "^4.9.5",
        "yargs": "^17.7.1"
    },
```

are related to tool `java-ts-gen` and on runtime `java-bridge` has no external dependencies?

Have you considered creating a separate npm module for `java-ts-gen` tool?
This would allow you to install it as dev dependency in the target project and not dragging baggage to production dependencies.

**Comment by MarkusJx** [comment]
`glob` is used in the main module in order to find the java dependency, but all other dependencies are just used for the typescript definition generator.

Moving that to a different module seems like a good Idea, I've created a new issue for this: #65

**Reply by MarkusJx** [reply]
Small update: With https://github.com/MarkusJx/node-java-bridge/pull/87, `glob` has been removed as a dependency.
With the next release, the module won't have any runtime dependencies.
